src/MessagePack.Experimental/MessagePack.Experimental.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
+    <TargetFrameworks>net8.0</TargetFrameworks>
```

Comment:
I would typically not be thrilled by this, but since it's in the Experimental package, I have no concerns.

Reply:
No, no.
Please be assured that I will not remove `net6.0` target framework in the later PR.
Except for `PrimitiveArrayResolver` being marked as `Obsolete` in .NET 8, the `Experimental` package will be unchanged.

The reason for removing `net6.0` in this PR draft is that it is tedious to write `#if` statements over and over.
In .NET 8, I wanted to concentrate on this work.